### ISSUE TITLE
Incomplete stack traces with non static dynamic imports

### ISSUE THREAD

**Issue by marvinhagemeister** [denoland/deno#20034]
When a dynamic import is not statically analysable we run into a situation where the stack trace is incomplete when an error is thrown. The stack trace seems to stop at the first non static import.

## Steps to reproduce

1. Clone https://github.com/marvinhagemeister/deno-dynamic-import-error
2. Run `deno run -A main.ts`

Actual error:

```sh
error: Uncaught (in promise) Error: No such file or directory (os error 2): readfile '/project/non-existant.json' at ext:deno_fs/30_fs.js:749:18

await import(url);
^
    at async file:///project/main.ts:2:1
```

Expected error:

```sh
error: Uncaught (in promise) Error: No such file or directory (os error 2): readfile '/project/non-existant.json'

await Deno.readTextFile(new URL("non-existant.json", base));
^
    at async Object.readTextFile (ext:deno_fs/30_fs.js:749:18)
    at async start (file:///project/thrower.ts:3:5)
    at async file:///project/indirect.ts:3:1
    at async file:///project/main.ts:2:1
```

**Comment by nayeemrmn** [comment]
Fixed in #20135.

**Comment by kt3k** [comment]
Confirmed it's fixed

```
$ ~/denoland/deno/target/debug/deno run -A main.ts
Stack trace should point to Deno.readTextFile
error: Uncaught (in promise) NotFound: No such file or directory (os error 2): readfile '/Users/kt3k/tmp/deno-dynamic-import-error/non-existant.json'
    await Deno.readTextFile(new URL("non-existant.json", base));
    ^
    at async Object.readTextFile (ext:deno_fs/30_fs.js:749:18)
    at async start (file:///Users/kt3k/tmp/deno-dynamic-import-error/thrower.ts:3:5)
    at async file:///Users/kt3k/tmp/deno-dynamic-import-error/indirect.ts:3:1
```
